.dprint.jsonc

```diff
@@ -0,0 +1,32 @@
+{
+  "exec": {
+    "commands": [{
+      "command": "rustfmt",
```

Comment:
the [`dprint` exec plugin](https://github.com/dprint/dprint-plugin-exec/) works with commands that takes STDIN or file paths as input.
So this is not compatible with `nargo fmt` (that command has no effect for me anyway).

So I am using `rustfmt` directly.